src/deluge/gui/ui/rename/rename_drum_ui.h

```diff
@@ -24,21 +24,15 @@ class SoundDrum;
 
 class RenameDrumUI final : public RenameUI {
 public:
-	RenameDrumUI();
-	bool opened() override;
-	ActionResult buttonAction(deluge::hid::Button b, bool on, bool inCardRoutine) override;
-	ActionResult padAction(int32_t x, int32_t y, int32_t velocity) override;
-	ActionResult verticalEncoderAction(int32_t offset, bool inCardRoutine) override;
-	bool getGreyoutColsAndRows(uint32_t* cols, uint32_t* rows) override;
-
-	// ui
-	bool exitUI() override;
+	RenameDrumUI(const char* title_) : RenameUI(title){};
```

Comment:
```suggestion
	RenameDrumUI(const char* title_) : RenameUI(title_){};
```

Reply:
If you're feeling brave, making the member in QwertyUI a `string_view` and renaming it to `title_` so it matches the convention we're trying to use might be better.

Reply:
> If you're feeling brave, making the member in QwertyUI a `string_view` and renaming it to `title_` so it matches the convention we're trying to use might be better.

I'm a bit confused. Is the convention to name initializer arguments `foo_` and members `foo`, or the other way around?